sdk/src/protocols/kelp/index.ts

```diff
@@ -0,0 +1,103 @@
+import { allow } from "zodiac-roles-sdk/kit"
+import { contracts } from "../../../eth-sdk/config"
+import { NotFoundError } from "../../errors"
+
+const tokens = [
+  {
+    name: "stETH",
+    address: contracts.mainnet.lido.stEth,
+  },
+  {
+    name: "ETHx",
+    address: contracts.mainnet.kelp.ethx,
+  },
+  {
+    name: "ETH",
+    address: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+  },
+]
+
+export const eth = {
+  stake: async ({
+    targets,
+  }: {
+    targets: ((typeof tokens)[number]["name"] | { address: `0x${string}` })[]
```

Comment:
let's restrict to only the supported list of addresses:

```suggestion
    targets: ((typeof tokens)[number]["name"] | (typeof tokens)[number]["address"])[]
```